scripts/user/add-judge.ts

```diff
@@ -0,0 +1,137 @@
+ *
+ *  Example usage:
+ *
+ * $ npx ts-node --transpile-only add-judge.ts Way "Kashi Way" judge.way@ustaxcourt.gov ["(123) 123-1234" false]
+ *
+ * Note that this script SHOULD be temporary: it is meant as a slight improvement from the current ill-defined process.
+ * Please extract into application logic!
+ */
+
+requireEnvVars(['ENV']);
+
+// eslint-disable-next-line @typescript-eslint/no-floating-promises
+(async () => {
+  const applicationContext = createApplicationContext();
+
+  // Required
+  const name = process.argv[2];
+  const judgeFullName = process.argv[3];
+  const email = process.argv[4];
+
+  // Optional, though a phone number needs to be provided eventually
+  // otherwise the chambers will have no associated phone number on trial info
+  const phone = process.argv[5];
+  const isSeniorJudge = process.argv[6]
+    ? process.argv[6].toLowerCase() === 'true'
+    : false;
+
+  // Check for mistaken emails
+  if (!emailIsInExpectedFormat({ email, judgeName: name })) {
+    const userInput = await promptUser(
+      `Warning: The email you entered does not match expected formats: ${expectedEmailFormats(name).join(', ')}. Continue anyway? y/n `,
+    );
+    if (userInput.toLowerCase() !== 'y') {
+      return;
+    }
+  }
+
+  // Check for mistaken phone numbers
+  if (phone && !phoneIsInExpectedFormat(phone)) {
+    const userInput = await promptUser(
+      'Warning: The phone number you entered does not match the expected format: (XXX) XXX-XXXX. Continue anyway? y/n ',
+    );
+    if (userInput.toLowerCase() !== 'y') {
+      return;
+    }
+  }
+
+  console.log('Setting up information to store ... ');
+  const section = getChambersNameFromJudgeName(name);
+  const judgeTitle = 'Judge';
+  const role = 'judge';
+  const userId = applicationContext.getUniqueId();
+
+  const userPoolId = await getUserPoolId();
+
+  environment.userPoolId = userPoolId;
+  const { tableName } = await getDestinationTableInfo();
+  environment.dynamoDbTableName = tableName;
+
+  const cognitoUserInfo: {
+    email: string;
+    role: Role;
+    name: string;
+    userId: string;
+    poolId?: string;
+    temporaryPassword?: string;
+    sendWelcomeEmail: boolean;
+  } = {
+    email,
+    name,
+    poolId: userPoolId,
+    role,
+    sendWelcomeEmail: true,
+    userId,
+  };
+
+  let dynamoUserInfo: RawUser = {
+    email,
+    entityName: 'User',
+    isSeniorJudge,
+    judgeFullName,
+    judgeTitle,
+    name,
+    role,
+    section,
+    userId,
+  };
+
+  if (phone) {
+    dynamoUserInfo = { ...dynamoUserInfo, contact: { phone } };
+  }
+
+  console.log('Adding user information to Cognito ... ');
+  await applicationContext
+    .getUserGateway()
+    .createUser(applicationContext, cognitoUserInfo);
+
+  console.log('Adding user information to Dynamo ... ');
+  const rawUser = new User(dynamoUserInfo).validate().toRawObject();
+  await applicationContext.getPersistenceGateway().createUserRecords({
```

Comment:
Maybe switch over to createOrUpdateUser() in admin.ts as it is how most every user gets created for scripts, test environments, and zenHub